app/services/permission.rb

```diff
@@ -18,7 +22,39 @@ def allow?
       registered_user_permissions
     else
       guest_user_permissions
+    end
+  end
+
+  def platform_admin_permissions
+    true
+  end
+
+  def campaign_manager_permissions
+    candidate = current_user.user_roles.first.candidate_id
+    # if current_path == "admin/#{candidate} then return true"
+    # unless candidate_id does not match the candidate that the manager is managing
+  end
+
+  def registered_user_permissions
+    return true if controller == "sessions"
+    return true if controller == "candidates"  && action.in?(%w(index show))
```

Comment:
Unnecessary spacing detected.<br>Operator `&&` should be surrounded with a single space.